simple-haskell-marker/src/main/java/alabno/simple_haskell_marker/HaskellSplitter.java

```diff
@@ -136,5 +145,30 @@ boolean isEmpty(String line) {
         }
         return true;
     }
+    
+    private HaskellBlock tryGet(List<HaskellBlock> blocks, int index) {
+        try {
+            return blocks.get(index);
+        } catch (IndexOutOfBoundsException e) {
+            return null;
+        }
+    }
+
+    public String getBlockAt(int lineNumber) {
```

Comment:
Name of function suggests it's a HaskellBlock being returned - maybe have the `getBlockText()` invoked elsewhere to make explicit it's a string or a block that's being returned.